python-avd/tests/models.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+import json
+from copy import deepcopy
+from functools import cached_property
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from ansible.inventory.manager import InventoryManager
+from ansible.parsing.dataloader import DataLoader
+from ansible.vars.manager import VariableManager
+from yaml import CSafeLoader, load
+
+from pyavd import get_avd_facts
+
+if TYPE_CHECKING:
+    from ansible.inventory.host import Host as AnsibleHost
+
+REPO_ROOT = Path(__file__).parents[2]
+MOLECULE_PATH = REPO_ROOT / "ansible_collections/arista/avd/molecule"
+
+
+class MoleculeHost:
```

Comment:
probably worth a bit of docstrings around here

Reply:
added